starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,41 @@
 // Iteration 1
 // 1. Create an array of strings
 
+const toDoList = [];
+
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
 
+function addTask(task: string) {
```

Comment:
```function addTask(task: string): number {```

Reply:
Remember that you can add types to the function. This one should only be able to return a number.